includes/Ajax.php

```
@@ -243,7 +248,64 @@ public function hide_pro_notice() {
+    /**
+     * Load more docs for DocsGrid widget AJAX pagination
+     *
+     * @since 2.1.12
+     *
+     * @return void
+     */
+    public function load_more_docs() {
+        check_ajax_referer('wedocs_load_more', 'nonce');
+
+        $page = isset($_POST['page']) ? intval($_POST['page']) : 1;
+        $widget_id = isset($_POST['widget_id']) ? sanitize_text_field($_POST['widget_id']) : '';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$widget_id'. (undefined)

(UnusedLocalVariable)